conjur/api/api.py

```diff
@@ -63,7 +64,8 @@ def __init__(self,
 
         self._default_params = {
             'url': url,
-            'account': account
+            'account': account,
+            'service_id': service_id
```

Comment:
I don't think `service_id` **_needs_** to be part of the `_default_params` object because it should doesn't need to be sent in every request. You can also just append it to the object during `login_ldap`. Your call though